pkg/plugins/scorer/kvcache_aware.go

```diff
@@ -13,41 +13,91 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/plugins"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/framework"
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/framework/plugins/multi/prefix"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/types"
 	logutil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/logging"
 )
 
-const (
-	// KvCacheAwareScorerType is the type of the KvCacheAwareScorer
-	KvCacheAwareScorerType = "kvcache-aware-scorer"
+// PrefixCachePluginMode defines the mode of the prefix cache plugin. It can be either `estimate` or `cache_tracking`.
+type PrefixCachePluginMode string
 
-	kvCacheRedisEnvVar     = "KVCACHE_INDEXER_REDIS_ADDR"
+const (
+	// PrefixCachePluginModeEstimate is the mode where the plugin use estimated prefix.
+	PrefixCachePluginModeEstimate PrefixCachePluginMode = "estimate"
```

Comment:
Q: do we still want environment based configuration or all via config file?
Might be a slippery slope adding more and more environment over time.